jaxrs-spec/src/main/asciidoc/chapters/filters_and_interceptors/_exceptions_filters_and_interceptors.adoc

```diff
@@ -17,7 +17,8 @@
 When a filter or interceptor method throws an exception, the server
 runtime will process the exception as described in
 <<exceptions_providers_server>>. As explained in
-<<exceptionmapper>>, an application can supply exception mapping
+<<exceptionmapper>>, a JAX-RS implementation will supply a default
```

Comment:
Not necessarily an issue for this request, but more for discussion with the community.  Are we allowed to use the term, "JAX-RS" in the spec?  This PR uses it a couple of places, but it looks like there are other places in the spec where we still use it.

Reply:
Right, I thought we have agreed to use the full Jakarta RESTful Web Services instead of the acronym to prevent any misuse.

Reply:
My understanding is that we can use "JAX-RS", but we agreed to move away from it. As said, the spec still uses the acronym in multiple places, so the PR should be fine.

Reply:
Yes, we are officially allowed to use JAX-RS as long as we want. Oracle's lawyers confirmed that is no Oracle trademark.